### ISSUE TITLE
Install issues

### ISSUE THREAD

**Issue by sstimeare** [Axadiw/MFIGamepadFeeder#11]
I've downloaded the MFIgamepadfeeder zip.  Then unzipped and located the app icon under C:\Users\Owner\AppData\Local\Apps\2.0\TE7H1C4G.YQ5\QW724811.V3L\mfig..tion_7140289108f10317_0001.0000_c9e771dd898da1a4 

when I try to open the app nothing happens.
I've deleted files manually as there's no uninstall and re-unzipped and re-installed but with the same results.  Please help.

**Comment by CarlosVAzevedo** [comment]
The same here :(

**Comment by dstryckman** [comment]
I'm having the same issue. However, I did see that a process is being created and that this process can be killed through the task manager. This process is described as being a 32 bits process in the list, while my version of Window 10 is a 64bit, Could that be an issue? Also, when I start the .exe file, something pops in the bar. I've been able to maxims this pop and all I get is a blank black UI. Any suggestions?
